_pages/resources.md

```diff
@@ -1,6 +1,7 @@
 ---
 permalink: /resources/
 title: Guidance & Resource Materials
+description: "Detailed guidance documents explaining the rights and responsibilities under the ADA and the Department's regulations related to topics the ADA covers including service animals, health care and more."
```

Comment:
I'm not sure if it matters for these frontmatter fields, but should this not have quotes?

```suggestion
description: Detailed guidance documents explaining the rights and responsibilities under the ADA and the Department's regulations related to topics the ADA covers including service animals, health care and more.
```

Reply:
some of them do, some of them don't weirdly but I think you're right that it's not necessary